<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <link rel="stylesheet" href="styles.css">
  <link rel="stylesheet" href="../styles.css">
  <title>Document</title>
</head>
<body>
  <nav class="main-nav">
    <ul class="main-ul">
      <li><a href="../index.html">HOME</a></li>
      <li><a href="index.html">PROJECT</a></li>
      <li><a href="../exp/index.html">EXPERIMENTS</a></li>
      <li><a href="../notebook/index.html">NOTEBOOK</a></li>
      <li><a href="../parts/index.html">PARTS</a></li>
      <li><a href="../team/index.html">TEAM</a></li>
      <li class="dropdown">
        <div>
          <a href="">MORE</a>
          <ul class="dropdown-content">
            <li><a href="../safety/index.html">SAFETY</a></li>
            <li><a href="../collab/index.html">COLLABORATIONS</a></li>
            <li><a href="../human/index.html">HUMAN PRACTICES</a></li>
            <li><a href="../modelling/index.html">MODELLING</a></li>
            <li><a href="../partimprov/index.html">PART IMPROVEMENT</a></li>
            <li><a href="../attribution/index.html">ATTRIBUTIONS</a></li>
          </ul>
        </div>
      </li>
    </ul>
  </nav>

  <div id="triangle"></div>
  <div class="project-section project-section-b" id="project-intro">
    <h1>TO THE CENTROMERE</h1>
    <h2>with retrotransposon VLPs</h2>
    <p>Centromere retrotransposon (CR) elements offer a natural transportation system to carry genes to the centromere. These elements
      utilize virus-like particles to encapsulate their genome for reverse transcription, then reintegration into the centromere.
      Centromeres as a transgene target allows for the accumulation of traits due to lack of recombinatorial events. We explored
      the virus-like particle vehicle in this system and measured the stability of the structure for packaging molecular cargo.</p>
    <a href="#abstract">See our abstract.</a>
  </div>

  <div class="project-section project-section-w" id="project-section1">
    <h1>NATURAL VLP FORMATION</h1>
    <h2>using bioinformatics and a protease assay</h2>
    <p>To determine the most natural construct for VLP formation, we needed to identify the protease cut site that would allow for mature VLP formation. Initial research on aspartyl proteases allowed us to determine putative protease cleavage sites between hydrophobic amino acid residues.</p>
    <p>To experimentally determine the protease cleavage sites, we transformed E.coli with a construct containing the gag and protease domains. Following induction, autocatalytic protease activity was observed and the resulting protein bands were sent to the Taplin Mass Spectrometry
    Facility. Tryptic fragments confirmed our previous putative cut site downstream of the nucleocapsid.</p>
    <a href="../exp/index.html">View our experiment.</a>
  </div>

  <div class="project-section project-section-w" id="project-section2">
    <h1>VLP STRUCTURE AND STABILITY</h1>
    <h2>through a combinatorial type assay</h2>
    <p>Constructs ending at the confirmed protease cleavage site, containing variations of the capsid, with N or C-terminal purification tags, and the presence or absence of extra amino acids were amplified. Purified proteins were subjected to various VLP assembly conditions and viewed under the electron microscope.</p>
    <a href="../exp/index.html">See our results.</a>
  </div>

  <div class="project-section project-section-w" id="project-section3">
    <h1>DETECTING VLP ASSEMBLY</h1>
    <h2>through a fluorescent protein attachment</h2>
    <p>Another construct containing our confirmed VLP forming sequence and a red fluorescent protein is designed. Expression of the sequence reveals. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima explicabo placeat enim voluptate fugit tempora aperiam exercitationem quisquam, adipisci dolor amet laboriosam aliquam quae nisi, a sed vel qui et?</p>
    <a href="../exp/index.html">See how.</a>
  </div>

  <div id="abstract">
    <h1>PROJECT ABSTRACT</h1>
    <p> &emsp;&emsp;&emsp;&emsp;&emsp; Nature has provided a remarkable system to insert genes into functional centromeres of grass genomes. Specifically, centromeric retrotransposons (CR) have the unique ability to insert themselves into the centromere by targeting a yet unidentified docking agent. We plan to adapt this system to insert genes of interest into centromeres. Centromeres are advantageous transgene targets because they lack recombination, allowing the stacking of multiple traits. Retrotransposons, or “jumping genes,” self-replicate and package their genome into self-assembling virus-like particles (VLPs), then reinsert (or “jump”) themselves into a new chromosomal location. To measure the stability of VLPs for packaging molecular cargo, we cloned the full-length gene encoding the CR gag protein and successfully generated VLPs in vitro. We also tested the efficiency of different gene constructs in forming VLPs in vitro. Electron microscopy can confirm VLP assembly, however, we plan to develop a convenient fluorescent assay to assess VLP assembly.</p>
  </div>

  <div id="spacer-1"></div>

  <div id="spacer-2"></div>

  <!-- FOOTER -->
  <div id="footer">
    <div class="f-section">
      <h2>CONTACT</h2>
      <p>uhigem@hawaii.edu</p>
    </div>
    <div class="f-section">
      <h2>LOCATE</h2>
      <p>University of Hawaii at Manoa</p>
      <p>1955 East West Road</p>
      <p>Honolulu, HI 96822</p>
    </div>
    <div class="f-section">
      <h2>NAVIGATE</h2>
      <div id="navigation">
        <div class="col">
          <a href="">Home</a>
          <a href="">Project</a>
          <a href="">Notebook</a>
        </div>
        <div class="col">
          <a href="">Parts</a>
          <a href="">Human Practices</a>
          <a href="">Safety</a>
        </div>
        <div class="col">
          <a href="">Collaboration</a>
          <a href="">Modelling</a>
          <a href="">Awards</a>
        </div>
      </div>
    </div>
  </div>

</body>
</html>
